### ISSUE TITLE
Enable additional static columns via config

### ISSUE THREAD

**Issue by tredman** [honeycombio/honeycomb-kubernetes-agent#42]
Sometimes it's useful to attach arbitrary fields to log events, such as cluster name or ID. We should support this by adding a section to the config to allow arbitrary key/value pairs to be appended to all events. Here's how it might look:

```yaml
watchers:
      - dataset: kubernetes-logs
        labelSelector: ""
        parser: glog
    verbosity: debug
# ....

additionalFields:
  - key: myField
    value: myValue
  - key: #...
```

**Comment by seh** [comment]
If you could accommodate the key name being a YAML [mapping key](https://yaml.org/spec/1.2/spec.html#mapping//), you could guarantee that they'd be unique.
```yaml
additionalFields:
  myField: myValue
```
Against that, though, you can't write a schema for "additionalFields" field beyond `map[string]string`. That's why the Kubernetes project favors the style you proposed. They leave the duplicate name problem to the consumer.

**Comment by tredman** [comment]
Included in 1.3.0
